hedera-node/test-clients/build.gradle.kts

```diff
@@ -230,6 +232,21 @@ val yahCliJar =
         }
     }
 
+val rcdiffJar =
+    tasks.register<ShadowJar>("rcdiffJar") {
+        exclude(listOf("META-INF/*.DSA", "META-INF/*.RSA", "META-INF/*.SF", "META-INF/INDEX.LIST"))
+        from(sourceSets["rcdiff"].output)
+        archiveFileName.set("rcdiff.jar")
+        configurations = listOf(project.configurations.getByName("rcdiffRuntimeClasspath"))
```

Comment:
@jjohannes Please take a look at this as well.

Reply:
This is okay. You could also use the `[...]` notation if you want to be consistent – `project.configurations["rcdiffRuntimeClasspath"]` – but that us just a different syntax.